### PR TITLE
Company showcase component

### PR DESCRIPTION
**Pull request by marwazpov**
this PR  is linked to issue #14

### CODE REVIEW

**Comment by marwazpov** [comment]
Hi @AllanSaleh , did anyone push into this by mistake?

**Comment by AllanSaleh** [comment]
Hi, @marwazpov Yes, I updated this branch to fix the test from failing and it worked. I will see why it failed again.

**Comment by AllanSaleh** [comment]
Hey @marwazpov Yes, I think someone pushed onto your branch. I will undo those changes.

**Review by AllanSaleh — CHANGES_REQUESTED** [review]
Great job on creating this component!!🙌

Just a couple of small changes to the design are required:
-  The paragraph has no space on the left and is sticking to the edge of the screen. I think adding some space to it will make it look better.
image
- The paragraph is aligned to the left still on mobile screens. If you can align it to the center on the mobile screen may make it look better.
- Maybe add some padding to the top and bottom of the component to give it some space from other components.
image

Other than these small tweaks, your component is coded great!👏

**Comment by marwazpov** [comment]
Hi @AllanSaleh , I have done the requested changes! I have also tried to make the component more reusable so I can use it in issue number 21. Let me know what you think ☺️

File: src/data/showCaseData.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    "amazon",
+    "microsoft",
+    "apple",
+    "google",
+    "airbnb",
+    "spotify",
+    "yahoo",
+    "netflix"
+]   
+}
```
Comment (AllanSaleh) [inline src/data/showCaseData.js:14]: Great work on separating the data from the component and putting it into its own file!

File: src/components/companyShowcaseComponent/CompanyShowcaseComponent.jsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+       <h2 className="font-inter text-blue-900 font-semibold self-end text-xs mt-5 mb-5 md:text-2xl col-span-5">
+        {showCaseData.header}</h2>
+        {topRowComanpies}
+      </div>
+      <div className="grid grid-cols-4 gap-x-5 place-content-center items-center w-5/6">
+       {bottomRowComanpies}
+      </div>
+     </section>
+  );
+}
```
Comment (AllanSaleh) [inline src/components/companyShowcaseComponent/CompanyShowcaseComponent.jsx:31]: AWESOME work on using props to make the component reusable! It is really well-coded!! This component can be used for different data now!! All thanks to you!🙌
And well done on using slice!!!👏

**Review by AllanSaleh — APPROVED** [review]
Superb work!!😍
Your component is now perfect!!🎊
Keep up the great work!!🔥

**Pull request merged**